tests/webscraping/test_listserv.py

```diff
@@ -9,13 +9,15 @@
 import bigbang
 from bigbang import listserv
 from bigbang.listserv import ListservArchive, ListservList, ListservMessage
+from config.config import CONFIG
 
+dir_temp = tempfile.gettempdir()
+project_directory = str(Path(os.path.abspath(__file__)).parent.parent.parent)
 url_archive = "https://list.etsi.org/scripts/wa.exe?"
 url_list = url_archive + "A0=3GPP_TSG_CT_WG6"
 url_message = url_archive + "A2=ind2101A&L=3GPP_TSG_CT_WG6&O=D&P=1870"
-dir_temp = tempfile.gettempdir()
 file_temp_mbox = dir_temp + "/listserv.mbox"
-file_auth = "../config/authentication.yaml"
+file_auth = project_directory + "/config/authentication.yaml"
```

Comment:
Paths to authentication files (or just to the config directory, if that's going to be used a lot) could be made available from `CONFIG`, like a path in `config.yml`, rather than relying on putting together these paths from strings.